src/main/kotlin/com/gaelmarhic/quadrant/helpers/ActivityFilteringHelper.kt

```diff
@@ -56,6 +59,24 @@ class ActivityFilteringHelper(
                 )
             }
 
```

Comment:
We should add unit tests for this new behaviour in order to make sure it works properly for both FQCN and PQCN.
You can create a new inner class for this, at the root level, just above the one called `GenerateByDefault`.

Reply:
Added unit tests

Reply:
Overall, the changes look good! There is just a couple of dummy things I'd like to change (my bad for not explaining them properly). In order to avoid more back and forth I will merge the PR and do these myself :)